### ISSUE TITLE
vendor Qt is outdated and doesnt work with PySide6

### ISSUE THREAD

**Issue by hannesdelbeke** [pyblish/pyblish-lite#133]
```
File "...\site-packages\pyblish_lite\vendor\Qt.py", line 1676, in _install
LogPython: Error:     raise ImportError("No Qt binding were found.")
LogPython: Error: ImportError: No Qt binding were found.
```

**Comment by mottosso** [comment]
Indeed, it'll need to pop in the new Qt.py and make a few tweaks to make it work. A PR would be lovely!

**Comment by aharjunmaa** [comment]
I was able to circumvent this error by simply copying the newest Qt.py module (v1.4.1) into the "vendor" directory to overwrite the old one. @mottosso, is this a sufficient solution?

**Comment by mottosso** [comment]
Yep, that's it!

**Comment by aharjunmaa** [comment]
Here's the PR:

https://github.com/pyblish/pyblish-lite/pull/134

I'm not sure exactly what "few tweaks" are necessary; I did find one PySide6 bug that I fixed as well, and after that, our custom UI that is based on Pyblish Lite works as before the PySide6 update. More extensive testing may be necessary.

**Comment by ArlonSwaders** [comment]
@mottosso the vendorized Qt.py is great for studio's and users using Pyblish Lite out of the box. However, we are using Rez to add Qt.py to our environment. In that situation, the vendorized `Qt.py` module is actually a maintenance burned when moving to for example Maya 2025, as we now would need the `vendor\Qt.py` module to be updated as well.

Do you have any concerns to add the vendorized module as a fallback for when `Qt.py` is not in the environment already?
```python
try:
    from Qt import QtCore
except ImportError:
    # Fall back to vendorized Qt.py
    from .vendor.Qt import QtCore
```

I'd be happy to open an PR for that.

**Comment by mottosso** [comment]
> the vendorized Qt.py module is actually a maintenance burned

It definitely should not be. The vendored Qt.py is an implementation detail of Pyblish Lite and should have no effect on anything outside of it. It's entirely safe to ignore from a pipeline perspective.

**Comment by ArlonSwaders** [comment]
@mottosso I'm not sure if that means you agree with the above suggestion or not?

---
To demonstrate the problem, I'm loading a Rez environment with the following:
  - maya-2025 (ships with PySide6)
  - qt_py-1.4.1
  - pyblish_lite-0.8.12

I run the following code in Maya to open the Pyblish Lite (normally we would use pyblish-maya):
```python
import pyblish.api
guis = pyblish.api.registered_guis()

for gui in guis:
    __import__(gui).show()
```

This results in the following error:
`# Error: ImportError: file /home/arlon.swaders/packages/pyblish_lite/0.8.13/site-packages/pyblish_lite/vendor/Qt.py line 1676: No Qt binding were found.`

This despite the fact that we have Qt.py-1.4 in our environment, which already is compatible with PySide6.

My point here is that, in order for us to move to Maya 2025, we need to wait for this repo (an Pyblish Maya) to update their vendorized Qt.py. This could in my opinion be slowing us down more than needed to roll out a new Maya version.

Hence, my suggestion to allow Pyblish Lite to pick up an external Qt.py, and fall back to the vendorized module when no external one is available.

**Comment by tihen7** [comment]
Our studio just encountered the same issue trying to run pyblish-lite in Deltagen 2025. We also load with rez and have tried different variations of QT/Pyside versions without success.

**Comment by mottosso** [comment]
PR is welcome!
